### DISCUSSION TITLE
ios客户端无法加载证书 (The ios client cannot load the certificate)

### DISCUSSION THREAD

**Discussion by omaidb** [slackhq/nebula#826] [Q&A]
我在Linux上为node创建好证书文件，crr文件和key文件，在ios客户端上加载时，ca文件加载正确，但是证书文件加载失败，是要把crt和key文件合并之后才能加载吗？

**Comment by johnmaguire** [comment]
> I have created the certificate file, crr file and key file for node on Linux. When loading on the ios client, the ca file is loaded correctly, but the certificate file fails to load. Should I combine the crt and key files before loading?

Can you describe what behavior you see when the certificate file fails to load?  Thanks.

**Comment by omaidb** [comment]
> 证书文件加载失败，

Prompt: Failed to load the certificate file。。。。。。
I've shut down the nebula server, I'll post a screenshot the next time I use it
